go.mod

```diff
@@ -88,6 +88,7 @@ require (
 	github.com/mitchellh/go-homedir v1.1.0
 	github.com/mitchellh/reflectwalk v1.0.1 // indirect
 	github.com/nwaples/rardecode v1.0.0 // indirect
+	github.com/onsi/ginkgo v1.13.0
```

Comment:
Please don't add this dependancy

Reply:
ginkgo is used for kubernetes end to end tests, and it provides asynchronous assertions which is desired, what kind of library does istio test infra provides which also provides asynchronous assertions?